src/main/webapp/i18n/de/exercise.json

```diff
@@ -75,6 +75,7 @@
             "assessmentDueDate": "Einreichungsfrist der Bewertung",
             "assessmentDateError": "Einreichungsfrist der Bewertung muss nach dem Veröffentlichungsdatum und nach der Einreichungsfrist für Aufgaben sein. Bitte überprüfe deine Auswahl nochmal",
             "dateNotSet": "Kein Datum gesetzt",
+            "sampleSolutionPublicationDate": "Veröffentlichungsdatum der Beispiellösung",
```

Comment:
We should ensure consistency here. The title in the exercise edit view follows this term (the title for the explanation below does the same). On the exercise detail page for instructors it is titled as "Beispiel Lösung". Same is the case for the exercise detail page for students.

Reply:
Thanks for the comments! I agree about consistency. These are the usages I found (frontend only), maybe I can change the two "Beispiel Lösung" usages to "Beispiellösung". I proposed this just by looking at the frequencies because my German is not good so I don't know which one would be better in terms of proper language usage, any further improvement suggestions for the German strings are welcome.

![image](https://user-images.githubusercontent.com/18427209/145687073-3072ba0f-dfb1-4296-9935-eb8b021ef7cd.png)
![image](https://user-images.githubusercontent.com/18427209/145687081-9a98e3d7-21db-4d78-94e4-da6cd8726282.png)

Reply:
About Example/Sample Solution distinction, "sample" seems to have more frontend usages but there are also considerable amount of "example" usages as well so I think I can change the "Example Solution Explanation" in this PR's screenshots to "Sample Solution Explanation" and maybe we can track the other usages in a new issue to not cause inconsistencies at other places. What do you think?

![image](https://user-images.githubusercontent.com/18427209/145689872-0abfa566-2a60-424c-b67f-105fa06acc25.png)
![image](https://user-images.githubusercontent.com/18427209/145689882-0e10265c-3b4f-4c67-bb23-8e4b2921532d.png)

Reply:
For me, "Beispiellösung" would be the preferred term as well. Regarding the Sample/Example solution issue, I'm totally fine with delaying this to another PR if many other files have to be changed in order to keep it consistent. This is not the scope of your PR, I would assume.